pkg/client/automation/client.go

```diff
@@ -267,3 +280,18 @@ func rmIDField(data *[]byte) error {
 	}
 	return nil
 }
+
+func get(a Client, resourceType ResourceType, offset int) (rest.Response, error) {
+
+	params := url.Values{}
+	params.Add("offset", strconv.Itoa(offset))
+
+	u, _ := url.ParseRequestURI(a.url)
```

Comment:
Was there a reason to use `ParseRequestURI`? 
I'd just use `url.Parse` instead, seeing as this URL does not actually fulfill the assumption that the 'url was received in an HTTP request'

Reply:
done